src/liveBlock.ts

```diff
@@ -1,21 +1,13 @@
 // ----- Imports ----- //
```

Comment:
It's still being used to parse the `LiveBlock`s for `Item`. I've left it in for a couple reasons:

1) I'm not sure whether this parsing code will live here or in `liveblog-rendering`
2) Removing it would require refactoring a much larger chunk of the codebase